modules/shapefile/docs/api-reference/shp-loader.md

```diff
@@ -0,0 +1,64 @@
+# SHPLoader
+
+<p class="badges">
+  <img src="https://img.shields.io/badge/From-v2.2-blue.svg?style=flat-square" alt="From-v2.2" />
+</p>
+
+Shapefile loader
+
+| Loader                | Characteristic                                |
+| --------------------- | --------------------------------------------- |
+| File Extension        | `.shp`,                                       |
+| File Type             | Binary                                        |
+| File Format           | [Well Known Binary][wkb]                      |
+| Data Format           | [Geometry](/docs/specifications/category-gis) |
+| Supported APIs        | `load`, `parse`, `parseSync`                  |
+| Decoder Type          | Synchronous                                   |
+| Worker Thread Support | Yes                                           |
+
+## Usage
+
+```js
+import {SHPLoader} from '@loaders.gl/wkt';
+import {parseSync} from '@loaders.gl/core';
+
+// prettier-ignore
+const buffer = new Uint8Array([
+  1, 1, 0, 0,   0,  0,  0,
+  0, 0, 0, 0, 240, 63,  0,
+  0, 0, 0, 0,   0,  0, 64
+]).buffer;
+const data = parseSync(buffer, SHPLoader);
+// => { positions: { value: Float64Array(2) [ 1, 2 ], size: 2 } }
+```
+
+```js
+import {SHPLoader} from '@loaders.gl/wkt';
+import {load} from '@loaders.gl/core';
+
+const data = await load(url, SHPLoader);
+```
+
+## Options
+
+N/A
+
+## Format Summary
+
+Well-known binary (SHP) is a binary geometry encoding to store geometries (it
```

Comment:
Is it WKB or not? maybe be a little more precise? Assume this section was copied and needs an update.

Reply:
Yes it needs an update.. I'll modify before landing